src/routes/authentication.js

```diff
@@ -6,18 +6,44 @@ const winston = require('winston'); // LOGGING
 const redis = require('../helpers/redisHelper');
 const unirest = require('unirest');
```

Comment:
Any particular reason you chose 'unirest'? I've been using 'axios' as http library and we should agree on one. According to npm statistics 'axios' is vastly more popular than 'unirest', which is usually a good sign